cohort-engine-api-web/src/test/resources/deploymentEnv.prop

```diff
@@ -0,0 +1,9 @@
+## setup the properties for the host under test
+## properties from this file will be picked up
+## if none are passed to the framework through
+## jvm arguments
+
+watson.service.hostname=localhost
+watson.service.port=9101
```

Comment:
Hard-coded port? Is this specific to your local Liberty setup or will it be the same for every consumer of the code?

Reply:
I will have to dig into this a bit more. It does get packaged with the tests jar that is intended to be used during the IBM Cloud toolchain deployment process, but unsure if it is actually used. It was pulled over from the example application.

Reply:
Talked with Tom P. and determined that this file is no longer used so I'm removing it.